elastic/apm-server
# Translate OpenTelemetry JVM metrics in processor/otel

## Motivation/summary

Remove the ingest pipeline which translates OpenTelemetry Java metrics, and do this in processor/otel instead. This ensures the copied metrics are grouped by their labels/dimensions, like all other metrics.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Run apm-server
2. Instrument a Java application with https://github.com/open-telemetry/opentelemetry-java-instrumentation, pointed at apm-server
3. Wait for metrics to be collected and sent to APM Server (not sure how long this is)
4. Navigate to the APM app in Kibana, and check the GC and memory usage graphs are populated
5. Check that the apm metrics docs are grouped into docs by labels

See also #4919

## Related issues

Closes https://github.com/elastic/apm-server/issues/5275
